datashuttle/utils/utils.py

```diff
@@ -9,11 +13,30 @@
 # --------------------------------------------------------------------------------------------------------------------
 
 
-def message_user(message: Union[str, list]) -> None:
+def log(message: str) -> None:
+    logger = logging.getLogger("fancylog_")
+    logger.info(message)
```

Comment:
I think I'm misunderstanding something, but why is everything logged as `INFO`? Typically I log at one of the [existing levels](https://docs.python.org/3/library/logging.html#logging-levels) (creating new ones if really necessary), then you can control what info goes where based on the level of severity. I.e. users typically only need to see high-level details in the console (`INFO`), but the developer probably wants as many details as possible in the log (`DEBUG`).

Reply:
thanks response as above will fix